server/simulation/random.js

```diff
@@ -32,6 +32,14 @@ const randomRating = () => {
   return parseFloat(rating);
 };
 
+/**
+ *   
+ * Returns an object with two properties: one number between 4 and 89, and one number between 2 and 9 
```

Comment:
I would perhaps use the text description here (which is for people) to describe the meaning of the function and leave the description of the structure of the returned object (which is for machines) to the `@returns` bit on the bottom.

For example:
````
/**
 * Returns an object with two properties containing a random number of missions completed, and
 * a smaller random number of missions completed in the last 7 days.
````